src/Ad5001/BetterGen/populator/CavePopulator.php

```diff
@@ -91,7 +92,7 @@ public function generateCave($x, $y, $z, Random $random) {
 		foreach($gen = $this->generateBranch($x, $y, $z, 5, 3, 5, $random ) as $v3 ) {
 			$generatedBranches --;
 			if ($generatedBranches <= 0) {
-				$gen->send(self::STOP);
+				$gen->send(self::STOP); // send not found.. @Ad5001 what is that
```

Comment:
@thebigsmileXD @jasonwynn10 http://php.net/manual/en/generator.send.php
It was my first generator.